bait-files/src/main/webapp/results.js

```diff
@@ -0,0 +1,124 @@
+const USERNAME_INPUT = 'username-input-container';
+const EMAIL_INPUT = 'email-input-container';
+const IMAGE_INPUT = 'image-input-container';
+const ADDRESS_INPUT = 'address-input-container';
+const NOTES_INPUT = 'notes-input-container';
+
+const PHONE_RESULTS = 'phone-results-container';
+const NAME_RESULTS = 'name-results-container';
+const USERNAME_RESULTS = 'username-results-container';
+const EMAIL_RESULTS = 'email-results-container';
+const IMAGE_RESULTS = 'image-results-container';
+const ADDRESS_RESULTS = 'address-results-container';
+const REVIEWER_NOTES = 'reviewer-notes-container';
+
+const VALID = 'VALID';
+const INVALID = 'INVALID';
+const NOT_PROVIDED = 'This information was not provided.';
+
+const invalidString =
+    'There were some problems validating the provided information.';
+const validString =
+    'There were no problems validating the provided information.';
+
+function getReview() {
+  const queryString = window.location.search;
+  const params = new URL(location.href).searchParams;
+  const requestId = params.get('reviewId');
+  if (requestId == null || isNaN(requestId)) {
+    window.location.replace('/');
+    return;
+  }
+
+  fetch('/review' + queryString)
+      .then(response => response.json())
+      .then((result) => {
+        document.getElementById(NAME_INPUT).appendChild(
+            document.createTextNode(checkInput(result.request.name)));
+        document.getElementById(EMAIL_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.email)));
+        document.getElementById(USERNAME_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.username)));
+        document.getElementById(PHONE_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.phoneNum)));
+        document.getElementById(ADDRESS_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.address)));
+        document.getElementById(IMAGE_INPUT)
+            .appendChild(
+                document.createTextNode(checkInput(result.request.image)));
+
+        var stars = ' '
+        for (let i = 0; i < MAX_RATING; i++) {
+          if (i < result.review.authenticityRating) {
+            stars += STAR_FILLED;
+            continue;
+          }
+          stars += STAR_OUTLINE;
+        }
+
+        document.getElementById(STAR_RATING)
+            .appendChild(document.createTextNode(stars));
+
+        if (result.review.authenticityRating <= 2) {
+          document.getElementById(AUTHENTICITY_EXPLANATION)
+              .appendChild(document.createTextNode(DOES_NOT_EXIST));
+        } else {
+          document.getElementById(AUTHENTICITY_EXPLANATION)
+              .appendChild(document.createTextNode(EXISTS));
+        }
+
+        document.getElementById(NAME_RESULTS)
+            .appendChild(document.createTextNode(
```

Comment:
very small suggestion: could maybe make a helper method for this?

Reply:
I agree that this might be a good idea.